nodes/src/nodes/ml_sklearn/IGlobal.py

```
@@ -0,0 +1,17 @@
+from .code import PreProcessor
+
+class IGlobal:
+    def __init__(self, IEndpoint):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)